src/form/tp-form.ts

```diff
@@ -58,6 +58,20 @@ export class TPFormElement extends HTMLElement {
 		// If form is valid then dispatch a custom 'submit-validation-success' event.
 		if ( formValid ) {
 			this.dispatchEvent( new CustomEvent( 'submit-validation-success', { bubbles: true } ) );
+
+			// If reset form is enabled then reset the form.
+			if ( 'yes' === this.getAttribute( 'submit-reset' ) && this.form ) {
+				// Reset form validation.
```

Comment:
@Takshil-Kunadia Lets create a new function called `reset` and move all this functionality there